src/views/LocationInputForm.jsx

```diff
@@ -318,16 +318,14 @@ export const LocationInputForm = connect(
                         value: e.target.checked,
                       });
                     }}
-                  />
-                  <Label
-                    className="msab-has-text-grey margin-left-10"
-                    htmlFor="tos"
                   >
-                    I agree to the{' '}
-                    <a href="tos" target="_new">
-                      terms of service
-                    </a>
-                  </Label>
+                    <span className="msab-has-text-grey margin-left-10">
```

Comment:
Ah, so you did need this after all!

Reply:
Yeah - I saw that this was what the category checkboxes were doing.